webapi/src/main/scala/org/knora/webapi/responders/admin/PermissionsResponderADM.scala

```diff
@@ -459,13 +465,15 @@ class PermissionsResponderADM(responderData: ResponderData) extends Responder(re
     /**
      * Gets a single administrative permission identified by project and group.
      *
-     * @param projectIri the project.
-     * @param groupIri   the group.
+     * @param projectIri        the project.
+     * @param groupIri          the group.
+     * @param requestingUser    the requesting user.
      * @return an [[AdministrativePermissionForProjectGroupGetResponseADM]]
      */
-    private def administrativePermissionForProjectGroupGetRequestADM(projectIri: IRI, groupIri: IRI, requestingUser: UserADM): Future[AdministrativePermissionForProjectGroupGetResponseADM] = {
-
-        // FIXME: Check user's permission for operation (issue #370)
+    private def administrativePermissionForProjectGroupGetRequestADM(projectIri: IRI,
+                                                                     groupIri: IRI,
+                                                                     requestingUser: UserADM
+                                                                    ): Future[AdministrativePermissionForProjectGroupGetResponseADM] = {
```

Comment:
Still needs a check if the user is project admin in the project that the permission is about.

Reply:
There is a validation for `AdministrativePermissionForProjectGroupGetADM` in messages. Since this responder calls `administrativePermissionForProjectGroupGetADM` that uses `AdministrativePermissionForProjectGroupGetADM`, I believe we do not need to define extra validation for `AdministrativePermissionForProjectGroupGetRequestADM `